### ISSUE TITLE
JHESSE Appreciation Thread

### ISSUE THREAD

**Issue by sibblegp** [CSSEGISandData/COVID-19#2232]
I wanted to first say thank you for collecting this data, allowing for incredible transparency into this pandemic.

And I wanted to say thank you more for doing a better job of keeping your data consistent so that our apps, sites, and research can proceed without interruption.

Thanks everyone! Keep up the great work!

Leave a thanks of your own if you agree!

**Comment by PavelGorbanj** [comment]
Thank you CSSE team!

**Comment by gsjennings** [comment]
Agreed, this has been very helpful, and in a format that can easily be automated to update with what I am tracking for my company.

**Comment by BaronBlakley** [comment]
Many thanks to everyone helping to put this together!

**Comment by lojic** [comment]
I am appreciative, but I'm also finding more and more errors in the data which is shaking my confidence in it. It would be nice if they were more open about their process - both in terms of scraping data sources, and in resolving errors.
